lib/hanami/action/callbacks.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require 'hanami/utils/class_attribute'
+require 'hanami/utils/callbacks'
```

Comment:
Should we also verify that Hanami's version is 1.3?

Reply:
added sir (was thinking about whether we should crash the app when Hanami version is different from 1.3, e.g. 2.0, but thinking it would be hinder deployment, so settle with just warning here)

Reply:
It probably won't hinder deployment because the test won't pass to begin with - and the PR for this won't get merged.

Reply:
make sense, updated sir!